src/tests/JIT/opt/Compares/compares.cs

```diff
@@ -82,7 +82,7 @@ public class SideEffects
     [MethodImpl(MethodImplOptions.NoInlining)]
     public static void Eq_byte_consume(byte a1, byte a2) {
         //ARM64-FULL-LINE: cmp {{w[0-9]+}}, {{w[0-9]+}}
-        //ARM64-FULL-LINE-NEXT: csel {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, eq
+        //ARM64-FULL-LINE-NEXT: csel {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, {{eq|ne}}
```

Comment:
how does the x64 validation works for these examples?

Reply:
It didn't do any disasm checks on them for x64. Added that now.